optaplanner-docs/src/modules/ROOT/pages/planner-configuration/planner-configuration.adoc

```diff
@@ -902,25 +918,11 @@ Planning list variable can be used if the domain meets the following criteria:
 . Each planning value is assigned to exactly one planning entity.
 No planning value may appear in multiple entities.
 
-[WARNING]
-====
-Planning list variable is a new feature and does not yet support the following advanced features:
-
-- overconstrained planning,
-- pinning,
-- value strength comparison,
-- nearby selection,
-- construction heuristic phase configuration,
-- exhaustive search,
-- partitioned search,
-- coexistence with another list or basic planning variable.
-====
-
 
 [[chainedPlanningVariable]]
 ==== Chained planning variable (TSP, VRP, ...)
 
-Some use cases, such as TSP and Vehicle Routing, require __chaining__.
+Some use cases, such as TSP and vehicle routing, require __chaining__.
```

Comment:
Do they? Is "chaining" the aggregate name we use for both "chained variable" and "list variable"? I am a bit confused now.

Reply:
Good catch! With the list variable, this is no longer true. It's now possible to implement VRP without chaining.

In my opinion, "chaining" should be used exclusively for `@PlanningVariable(graphType=CHAINED)`, and not as an aggregate for bot chained and list variables.

Reply:
I'm going to open another PR that addresses this line together with other parts of the documentation that became when the list variable was merged.